packages/mobile/components/drawers/send/views/AmountView.svelte

```diff
@@ -99,7 +99,7 @@
         updateNewTransactionDetails({
             type: $newTransactionDetails.type,
             rawAmount,
-            unit,
+            tokenMetadata: asset.metadata,
```

Comment:
we dont need to update mobile, i would also not add `tokenMetadata` into the transaction details